spec/features/admin_edits_user_profile_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+feature 'admin edits user profile', %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.